discord_bot.py

```diff
@@ -0,0 +1,304 @@
+
+        if is_everything_ok:
+            event.msg.reply('✅ Online multiplayer is working fine. Go play with others!')
+        else:
+            with app.app_context():
+                event.msg.reply('⚠️ Looks like online multiplayer is encountering issues. For details, head over here: {}'.format(url_for('online_multiplayer_status')))
+
+    def create_player_message_embed(self, player):
+        """Create a RWRS player rich Discord message."""
+        embed = self.create_base_message_embed()
+
+        embed.url = player.link_absolute
+        embed.title = 'Players › {} › {}'.format(player.database_name, player.username)
+
+        username_lower = player.username.lower()
+
+        if username_lower == app.config['MY_USERNAME']:
+            embed.description = ':wave: Hey, I\'m the creator of RWRS! Glad to see you\'re using this bot.'
+        elif username_lower in app.config['CONTRIBUTORS']:
+            embed.description = ':v: This player contributed in a way or another to RWRS. Thanks to her/him!'
+        elif username_lower in app.config['DEVS']:
+            embed.description = ':military_medal: Say hi to one of the Running With Rifles developers!'
+
+        embed.set_thumbnail(
+            url=player.rank.image_absolute
+        )
+
+        embed.add_field(
+            name='Current rank',
+            value='{}\n{} XP'.format(
+                player.rank.name,
+                helpers.humanize_integer(player.xp)
+            ),
+            inline=True
+        )
+
+        embed.add_field(
+            name='Next rank',
+            value='{}\n{} XP'.format(
+                player.next_rank.name,
+                helpers.humanize_integer(player.next_rank.xp)
+            ) if player.next_rank else 'Highest possible rank reached',
+            inline=True
+        )
+
+        if player.next_rank:
+            embed.add_field(
+                name='Next rank progression',
+                value='{}% - {} XP remaining'.format(
+                    player.xp_percent_completion_to_next_rank,
+                    helpers.humanize_integer(player.xp_to_next_rank)
+                )
+            )
+
+        embed.add_field(
+            name='Kills',
+            value=helpers.humanize_integer(player.kills),
+            inline=True
+        )
+
+        embed.add_field(
+            name='Deaths',
+            value=helpers.humanize_integer(player.deaths),
+            inline=True
+        )
+
+        embed.add_field(
+            name='K/D ratio',
+            value=player.kd_ratio,
+            inline=True
+        )
+
+        embed.add_field(
+            name='Score',
+            value=helpers.humanize_integer(player.score),
+            inline=True
+        )
+
+        embed.add_field(
+            name='Time played',
+            value=helpers.humanize_seconds_to_hours(player.time_played) + ' (' + helpers.humanize_seconds_to_days(player.time_played) + ')',
```

Comment:
Create a `Player` boolean attribute to know if the second time counter must be shown.